package-deps/README.md

```diff
@@ -0,0 +1,43 @@
+# @microsoft/package-deps
+
+`package-deps` is a general utility for building a JSON object containing the git hashes of all files used to produce a given package.
```

Comment:
The other thing is that when you put "deps" besides "package", people would typically interpret deps as "npm deps", so why not just package-hash and drop "deps" completely?